pom.xml

```diff
@@ -63,23 +63,23 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins.version>2.7.1</jenkins.version>
```

Comment:
Why not just stick with 2.7.1? That's the first LTS release of 2.x, it's what Declarative and BO depend on, we generally stick to depending on the earliest possible LTS release rather than a later one, etc. I know of no reason to depend on 2.7.3 rather than 2.7.1.